src/fastertransformer/th_op/multi_gpu_gptneox/ParallelGptNeoXOp.cc

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
Same to above, don't use `Parallelxxx` because we fuse multi gpu and single gpu in one implementation now.